Test/WebTestCase.php

```diff
@@ -119,7 +119,19 @@ protected function runCommand($name, array $params = array(), $reuseKernel = fal
         $input->setInteractive(false);
 
         $fp = fopen('php://temp/maxmemory:'.$this->maxMemory, 'r+');
-        $output = new StreamOutput($fp);
+
+        $commandVerbosity = StreamOutput::VERBOSITY_NORMAL;
+        if ($this->getContainer()->hasParameter('liip_functional_test.command_verbosity')) {
```

Comment:
should we not make these properties instead which are set with the configured defaults? this way we can override them on a per test basis